test/WhiteBoxTests.cpp

```diff
@@ -1955,6 +1957,45 @@ void WhiteBoxTests::testParseUrl()
                      net::parseUrl("https://sub.domain.com:80/some/path"));
 }
 
+void WhiteBoxTests::testSafeAtoi()
```

Comment:
Worth testing nullptr and 0 length combinations, also, null-terminated string with wrong length (should work because '\0' is not a digit).

Reply:
Good idea, done. :-)